### PR TITLE
Fixing postinstall script to use new Android libs

### PR DESCRIPTION
**Pull request by bhariharan**

### CODE REVIEW

**Comment by bhariharan** [comment]
@wmathurin

**Comment by wmathurin** [comment]
I added myself back as a watcher.

File: tools/postinstall-android.js [hunk 1]
```diff
@@ -120,7 +124,7 @@ var oldBuildScriptDepTree = "buildscript {";
 var newBuildScriptDepTree = "buildscript {\n\tdependencies {\n\t\tclasspath 'com.android.tools.build:gradle:1.3.1'\n\t}\n";
 shelljs.sed('-i', oldBuildScriptDepTree, newBuildScriptDepTree, path.join(appProjectRoot, 'build.gradle'));
 var oldLibDep = "compile \"com.android.support:support-v13:23+\"";
-var newLibDep = "compile project(':SmartSync')";
+var newLibDep = "compile \"com.android.support:support-v13:23+\"\ncompile project(':SalesforceHybrid')";
```
Comment (wmathurin) [inline tools/postinstall-android.js]: Why do we need the support lib?

Comment (bhariharan) [inline tools/postinstall-android.js]: ```Cordova``` needs it. We didn't need it earlier, because ```React``` pulled it in, so the ```APK``` had it.

Comment (wmathurin) [inline tools/postinstall-android.js]: Ok makes sense

**Comment by wmathurin** [comment]
:+1:

**Comment by bhariharan** [comment]
Also fixed plugin package paths [here](https://github.com/bhariharan/SalesforceMobileSDK-CordovaPlugin/commit/99061ab2f9e0e4df715c110609a01f7cb3bf2707).

**Pull request merged**